__tests__/database/schema/ChatSchema.test.ts

```diff
@@ -26,7 +27,8 @@ const savePrepReplacementExpectation = [
   'status',
   'createdAt',
   'updatedAt',
-  {}
+  {},
+  []
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, here `attachmentJson` is represented as an empty object `{}` and an empty array `[]`. It should be consistent with the actual data type used in the application.
```diff
-   {},
-   []
+   ''
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->